.vib/joomla/goss/joomla.yaml

```diff
@@ -0,0 +1,48 @@
+command:
+  check-app-version:
+    exec: grep {{ .Env.APP_VERSION }} /opt/bitnami/joomla/administrator/manifests/files/joomla.xml
+    exit-status: 0
+  # Placeholders in SQL installation files should have been substituted
+  sql-placeholders:
+    exec: grep "#__" /opt/bitnami/joomla/installation/sql/mysql/*.sql
+    exit-status: 1
+file:
+  # Joomla! should have been configured based on build-time defaults
+  /opt/bitnami/joomla/configuration.php:
+    exists: true
+    filetype: file
+    contains:
+      - "public $db = 'bitnami_joomla'"
+  # PHP settings should have been configured
+  /opt/bitnami/php/etc/php.ini:
+    exists: true
+    filetype: file
+    contains:
+      - /^memory_limit/
```

Comment:
Should we would add here the the value of `memory_limit` expected? Could we have this config param set with other value at some point? I ask because joomla's postunpack logic will add or replace the config values when using `php_conf_set()`

Reply:
👍 I have added the expected value that comes from `PHP_DEFAULT_MEMORY_LIMIT`:

https://github.com/bitnami/containers/blob/8338dbb4337cc1999cd73480d781e5d6699e6989/bitnami/joomla/4/debian-11/rootfs/opt/bitnami/scripts/joomla-env.sh#L117

If the value of the env-var is ever changed, the test will need to be updated. Is that what you mean and the expected behavior, right?

Reply:
Perfect! Yes, I think that is a valid tradeoff in this case.